Makefile

```diff
@@ -168,7 +168,10 @@ else ifdef DEPLOY_MANIFEST_TAG
   DEPLOY_TAG_OPTION = --deploy-manifest-tag "$(DEPLOY_MANIFEST_TAG)"
 endif
 
-deploy-all: $(BUILD_FOLDER) deploy-namespace deploy-postgres deploy-s3 deploy-ocm-secret deploy-route53 deploy-service
+_verify_minikube:
+	minikube status
+
+deploy-all: _verify_minikube $(BUILD_FOLDER) deploy-namespace deploy-postgres deploy-s3 deploy-ocm-secret deploy-route53 deploy-service
```

Comment:
We use deploy all in order to deploy to PSI as well.
Maybe it should in `deploy-test` and `jenkins-deploy-for-subsystem`?